network/omaha/protocol.go

```diff
@@ -253,7 +253,7 @@ type Action struct {
 	DisplayVersion        string `xml:"DisplayVersion,attr,omitempty"`
 	Sha256                string `xml:"sha256,attr,omitempty"`
 	NeedsAdmin            bool   `xml:"needsadmin,attr,omitempty"`
-	IsDeltaPayload        bool   `xml:"IsDeltaPayload,attr,omitempty"`
+	IsDelta               bool   `xml:"IsDelta,attr,omitempty"`
```

Comment:
No, `IsDeltaPayload` is the correct name. `IsDelta` was an older deprecated one which is why you may have seen it in an example but update_engine ignores it.